### DISCUSSION TITLE
Connect button (android) in Wallet Connect Error

### DISCUSSION THREAD

**Discussion by chengbrian** [Web3Auth/Web3Auth#603] [Help]
The button group in **Wallect Connect** is identified as "w3a-wallet-connect__container-ios" with the Metamask icon button – which is **_desired_** with a redirection link to Metamask. See below

![Screenshot 2022-09-13 at 2 41 57 PM](https://user-images.githubusercontent.com/82595666/189830422-3d722666-cb8b-4cd9-b9c2-2936bf46c84d.png)

However, when it's opened with _android_, the button group in **Wallect Connect** is identified as "w3a-wallet-connect__container-android". The connect buttons appear with no redirection to Metamask. 
![Screenshot 2022-09-13 at 2 41 37 PM](https://user-images.githubusercontent.com/82595666/189830556-ecd6eb30-2d92-4c22-b998-e67c141c3401.png)

Any idea on how to display the buttons as ios where users can redirect to Metamask?

**Comment by shahbaz17** [comment]
Hey @chengbrian 

Sorry for the delay in response. It looks like a slow internet issue. Can you try updating the package to the [latest version](https://github.com/Web3Auth/web3auth-android-sdk)? `v2.6.0`
